src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.TestFramework/KeyVaultTestFixture.cs

```diff
@@ -159,12 +169,39 @@ private bool FromConfiguration()
             return result.AccessToken;
         }
 
+        public async Task<string> GetUserAccessToken(string authority, string resource, string scope)
+        {
+            string clientId = TestConfigurationManager.TryGetEnvironmentOrAppSetting("NativeClientId");
+            var context = new AuthenticationContext(authority, tokenCache);
```

Comment:
@rijethma @RandalliLama this particular version of KeyVault nugets that are targeting .NET 4.5.2 will be taking dependency on NewtonsoftJson 6.0.8.
So make sure that it does not break anything
#3250